lenovo/legion/16ach6h/edid/default.nix

```diff
@@ -0,0 +1,14 @@
+{ config, pkgs, lib, ...}:
+
+let
+  chip_edid = pkgs.runCommandNoCC "chip_edid" { } ''
+    mkdir -p $out/lib/firmware/edid
+    cp ${./16ach6h.bin} $out/lib/firmware/edid/16ach6h.bin
```

Comment:
Where did you get this magic firmware file from?

Reply:
If Lenovo’s so-called “DDG" is turned off (hybrid), the built-in display will provide a different edid from the one with "DDG" turned on. This edid can only set the display to 60hz at the highest, so I just extracted the edid under "DDG" enable and used it. set as default

Reply:
Where is this term coming from? I was not able to find it online.

Reply:
To get the correct edid file, just do `cat /sys/class/drm/*/edid > 16ach6h.bin` with ddg enabled
If you say how to specify edid, you can refer to these links
https://discourse.nixos.org/t/how-to-tell-kernel-to-use-a-custom-modeline/5500
https://wiki.archlinux.org/title/Kernel_mode_setting#Forcing_modes_and_EDID
https://discourse.nixos.org/t/overwrite-lib-firmware-file/9650